internal/scheduler/database/models.go

```diff
@@ -56,18 +57,22 @@ type Queue struct {
 }
 
 type Run struct {
-	RunID        uuid.UUID `db:"run_id"`
-	JobID        string    `db:"job_id"`
-	Created      int64     `db:"created"`
-	JobSet       string    `db:"job_set"`
-	Executor     string    `db:"executor"`
-	Node         string    `db:"node"`
-	Cancelled    bool      `db:"cancelled"`
-	Running      bool      `db:"running"`
-	Succeeded    bool      `db:"succeeded"`
-	Failed       bool      `db:"failed"`
-	Returned     bool      `db:"returned"`
-	RunAttempted bool      `db:"run_attempted"`
-	Serial       int64     `db:"serial"`
-	LastModified time.Time `db:"last_modified"`
+	RunID               uuid.UUID          `db:"run_id"`
+	JobID               string             `db:"job_id"`
+	Created             int64              `db:"created"`
+	JobSet              string             `db:"job_set"`
+	Executor            string             `db:"executor"`
+	Node                string             `db:"node"`
+	Cancelled           bool               `db:"cancelled"`
+	Running             bool               `db:"running"`
+	Succeeded           bool               `db:"succeeded"`
+	Failed              bool               `db:"failed"`
+	Returned            bool               `db:"returned"`
+	RunAttempted        bool               `db:"run_attempted"`
+	Serial              int64              `db:"serial"`
+	LastModified        time.Time          `db:"last_modified"`
+	LeasedTimestamp     pgtype.Timestamptz `db:"leased_timestamp"`
+	PendingTimestamp    pgtype.Timestamptz `db:"pending_timestamp"`
+	RunningTimestamp    pgtype.Timestamptz `db:"running_timestamp"`
+	TerminatedTimestamp pgtype.Timestamptz `db:"terminated_timestamp"`
```

Comment:
the new fields here uses `pgytype.Timestamptz`. I tried to force switch it to `time.Time` so we become consistent with previous fields, but it appears that `sqlc` uses `pgtype.Timestamptz` for nullable time types.

Reply:
Is it not possible to override the default types? I remember seeing config for that n the past.

Reply:
https://docs.sqlc.dev/en/latest/howto/overrides.html

Reply:
yeah, I actually did the override here https://github.com/armadaproject/armada/pull/3046/files/f72838670aaf0eaadea7f079e451f53fb10211a7#diff-5d7662fd33b2fd535f7090bf737e198d36400a8597c740e8380b88ed9f4747f8R18-R19
but it didn't seem to have effect. trying something hope it works

Reply:
yeah it worked
```yaml
          - db_type: "timestamptz"
            go_type: "time.Time"
            nullable: true
          - db_type: "timestamptz"
            go_type: "time.Time"
```
one for nullable timestamps and other for the non-nullable ones